src/plugins/lua/known_senders.lua

```diff
@@ -197,6 +244,94 @@ local function known_senders_callback(task)
   end
 end
 
+local function verify_local_replies_set(task)
+  local replies_sender = task:get_reply_sender()
+  if not replies_sender then
+    lua_util.debugm(N, task, 'Could not get sender')
+    return nil
+  end
+
+  local replies_recipients = task:get_recipients('mime')
+
+  local replies_sender_string = lua_util.maybe_obfuscate_string(tostring(replies_sender), settings, settings.sender_prefix)
+  local replies_sender_key = make_key_replies(replies_sender_string:lower(), 8)
+
+  local function redis_zscore_script_cb(err, data)
+    if err ~= nil then
+      rspamd_logger.errx(task, 'Could not verify %s local replies set %s', replies_sender_key, err)
+    end
+    rspamd_logger.infox(task, 'DATA: %s', data)
+    if data ~= 1 then
+      rspamd_logger.infox(task, 'Recipients were not verified')
```

Comment:
`debugm` here please.

Reply:
In which cases should I use `debugm` instead of `infox`, so that I know this and do not make such a mistake?

Reply:
In all cases aside of the cases when there is some important information that must attract user's attention.